fog/ingest/server/src/config.rs

```diff
@@ -115,6 +115,10 @@ pub struct IngestConfig {
     /// Postgres config
     #[clap(flatten)]
     pub postgres_config: SqlRecoveryDbConnectionConfig,
+
+    /// How many milliseconds to wait between polling.
+    #[clap(long, default_value = "250", value_parser = parse_duration_in_millis, env = "MC_POLL_INTERVAL")]
```

Comment:
I'm undecided, should this have an `MS` suffix or similar. It looks like the other uses of MC_POLL_INTERVAL are seconds, so when one looks in the help charts or similar these values will seem different, it would be unfortunate for someone to set this to 1ms or 5ms on accident
```
➜  mobilecoin git:(fog-configurable-poll-interval) rg MC_POLL
watcher/src/config.rs
34:    #[clap(long, default_value = "1", value_parser = parse_duration_in_seconds, env = "MC_POLL_INTERVAL")]

fog/ingest/server/src/config.rs
120:    #[clap(long, default_value = "250", value_parser = parse_duration_in_millis, env = "MC_POLL_INTERVAL")]

fog/ledger/server/src/config.rs
165:    #[clap(long, default_value = "250", value_parser = parse_duration_in_millis, env = "MC_POLL_INTERVAL")]

mobilecoind/src/config.rs
67:    #[clap(long, default_value = "5", value_parser = parse_duration_in_seconds, env = "MC_POLL_INTERVAL")]
```

Reply:
I also think there might be some conflict with other places we use `POLL_INTERVAL` 

maybe `MC_DB_POLL_INTERVAL_MS` might be a good descriptor

Reply:
sounds good, i'll add a `_MS` suffix